pir/cpp/client_test.cpp

```diff
@@ -72,12 +77,52 @@ TEST_F(PIRClientTest, TestProcessResponse) {
   Context()->Encoder()->encode(value, pt);
   vector<Ciphertext> ct(1);
   Encryptor()->encrypt(pt, ct[0]);
-  PIRPayload payload = PIRPayload::Load(ct);
 
-  auto result = client_->ProcessResponse(payload).ValueOrDie();
+  Reply reply;
+  *reply.mutable_reply() = SaveCiphertexts(ct).ValueOrDie();
+
+  auto result = client_->ProcessResponse(reply).ValueOrDie();
   ASSERT_EQ(result, value);
 }
 
+TEST_F(PIRClientTest, TestPayloadSerialization) {
```

Comment:
Why is this here instead of with payload?

Reply:
Because the client does the serialization.